certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -504,16 +508,2532 @@
+						"title": "",
+						"default": "location",
+						"x-ms-visibility": "internal"
+					  }
+					},
+					"description": "content",
+					"required": [
+					  "contentType"
+					]
+				  }
+				},
+				"required": [
+				  "channel",
+				  "content"
+				]
+			  }
+			}
+		  ]
+		}
+	  },
+	  "/conversations/v3/power-automate/messages/whatsapp/quick-reply": {
+		"post": {
+		  "responses": {
+			"202": {
+			  "description": "default",
+			  "schema": {
+				"type": "object",
+				"properties": {
+				  "messageId": {
+					"type": "string",
+					"description": "messageId"
+				  },
+				  "timestamp": {
+					"type": "string",
+					"description": "timestamp"
+				  }
+				}
+			  }
+			}
+		  },
+		  "parameters": [
+			{
+			  "name": "body",
+			  "in": "body",
+			  "required": false,
+			  "schema": {
+				"type": "object",
+				"properties": {
+				  "from": {
+					"type": "string",
+					"description": "from"
+				  },
+				  "to": {
+					"type": "string",
+					"description": "to"
+				  },
+				  "channel": {
+					"type": "string",
+					"description": "channel",
+					"title": "",
+					"default": "whatsapp",
+					"x-ms-visibility": "internal"
+				  },
+				  "content": {
+					"type": "object",
+					"properties": {
+					  "contentType": {
+						"type": "string",
+						"description": "contentType",
+						"title": "",
+						"default": "interactive",
+						"x-ms-visibility": "internal"
+					  },
+					  "interactive": {
+						"type": "object",
+						"properties": {
+						  "subType": {
+							"type": "string",
+							"description": "subType",
+							"title": "",
+							"default": "buttons",
+							"x-ms-visibility": "internal"
+						  },
+						  "components": {
+							"type": "object",
+							"properties": {
+							  "header": {
+								"type": "object",
+								"properties": {
+								  "type": {
+									"type": "string",
+									"description": "type"
+								  },
+								  "text": {
+									"type": "string",
+									"description": "text"
+								  }
+								},
+								"description": "header"
+							  },
+							  "body": {
+								"type": "object",
+								"properties": {
+								  "type": {
+									"type": "string",
+									"description": "type"
+								  },
+								  "text": {
+									"type": "string",
+									"description": "text"
+								  }
+								},
+								"description": "body"
+							  },
+							  "footer": {
+								"type": "object",
+								"properties": {
+								  "type": {
+									"type": "string",
+									"description": "type"
+								  },
+								  "text": {
+									"type": "string",
+									"description": "text"
+								  }
+								},
+								"description": "footer"
+							  },
+							  "buttons": {
+								"type": "array",
+								"items": {
+								  "type": "object",
+								  "properties": {
+									"type": {
+									  "type": "string",
+									  "description": "type",
+									  "default": "reply",
```

Comment:
@ridlees The property is internal and has a default value, it must be required. Optional internal fields are ignored.

Reply:
Fixed